scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla/balrog:dockerhub"
```

Comment:
if coveralls is run on pull request, yes it will override the coverage for master. I think you can override that from coveralls.io , by going to the repo settings on coveralls and disabling Pull Request Comment.

<img width="724" alt="screen shot 2017-05-03 at 6 38 12 pm" src="https://cloud.githubusercontent.com/assets/25608335/25673573/cda8753a-302f-11e7-928b-754d72b20757.png">